smarts/core/sensors.py

```diff
@@ -901,11 +900,11 @@ def __init__(self, vehicle, sim, plan, horizon=32):
         self._plan = plan
         self._horizon = horizon
 
-    def __call__(self):
+    def __call__(self) -> RoadWaypoints:
         veh_pt = self._vehicle.pose.point
         lane = self._road_map.nearest_lane(veh_pt)
         if not lane:
-            return RoadWaypoints(lanes={}, route_waypoints=[])
```

Comment:
I suspect the reason for the redundancy here was so that users would only ever have to add one type of waypoints sensor, either `RoadWaypointsSensor` or `WaypointsSensor`, but not both.  I.e., I think this was just to simplify things for users.